app/components/course-page/introduction-step/create-repository-card/request-language-dropdown.js

```diff
@@ -68,7 +68,7 @@ export default class RequestLanguageDropdownComponent extends Component {
   @action
   handleInputDidInsert(element) {
     this.inputElement = element;
-    this.inputElement.focus();
+    this.inputElement.focus({ preventScroll: true });
```

Comment:
@libmartinito I'm not sure about changing this everywhere else - it makes sense for concepts since we **know** that scrolling is handled somewhere else. In these other cases I'm not sure if this is the case. Can we keep the other use-cases as-is for now? If we discover more scroll+focus interactions then we can think about how to handle them separately. Even for `focus-on-insert`, `preventScroll` should be an option - there are places where we'd want this and places where we wouldn't.